test/exception/NotYetImplementedTest.cpp

```diff
@@ -0,0 +1,41 @@
+/***********************************************************************/
+/*  Copyright 2015 ToPS                                                */
+/*                                                                     */
+/*  This program is free software; you can redistribute it and/or      */
+/*  modify it under the terms of the GNU  General Public License as    */
+/*  published by the Free Software Foundation; either version 3 of     */
+/*  the License, or (at your option) any later version.                */
+/*                                                                     */
+/*  This program is distributed in the hope that it will be useful,    */
+/*  but WITHOUT ANY WARRANTY; without even the implied warranty of     */
+/*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the      */
+/*  GNU General Public License for more details.                       */
+/*                                                                     */
+/*  You should have received a copy of the GNU General Public License  */
+/*  along with this program; if not, write to the Free Software        */
+/*  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,         */
+/*  MA 02110-1301, USA.                                                */
+/***********************************************************************/
+
+// External headers
+#include "gmock/gmock.h"
+
+// ToPS headers
+#include "exception/NotYetImplemented.hpp"
+
+using ::testing::Eq;
+
+using tops::exception::NotYetImplemented;
+
+void throwANotYetImplementedException() {
+  throw NotYetImplemented(__func__);
+}
+
+TEST(NotYetImplemented, ShouldThrowAMessage) {
+  try {
+    throwANotYetImplementedException();
+  } catch(NotYetImplemented& e) {
+    ASSERT_STREQ(e.what(),
+      "The method throwANotYetImplementedException is not yet implemented");
+  }
+}
```

Comment:
Try to separate your tests in sections, like the other tests. You can find all file separators (the funny comments) within `doc/GUARD.cpp`.